<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget">
      <property name="styleSheet">
       <string notr="true">QListWidget {
    background-color: #2b2b2b;  /* Темный фон для списка */
    border: none;
    border-right: 2px solid #444d58; /* Разделительная линия справа */
    font-size: 16px;
}

QListWidget::item {
    background-color: #2b2b2b;  /* Темный фон для каждого элемента */
    color: #d1d1d1;              /* Бело-серый текст */
    border-bottom: 1px solid #444d58; /* Линия между кнопками */
    padding: 10px;
    font-weight: bold;
    border-radius: 10px;
    transition: background-color 0.3s;
}

QListWidget::item:selected {
    background-color: #6fcf97;  /* Зеленый фон для выбранного элемента */
    color: #1e1e1e;             /* Тёмный текст на зеленом фоне */
}

QPushButton {
    background-color: #6fcf97;  /* Мягкий зеленый фон для кнопок */
    color: #1e1e1e;             /* Тёмный текст */
    font-size: 16px;
    font-weight: bold;
    border: none;
    padding: 12px 24px;
    border-radius: 25px;
    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #5bbd88;  /* Зеленый цвет на наведении */
    box-shadow: 0 6px 12px rgba(0, 0, 0, 0.2);
}

QPushButton:pressed {
    background-color: #4aa97a;  /* Темный зеленый при нажатии */
    box-shadow: none;
}

QPushButton:disabled {
    background-color: #3e6b4c;  /* Цвет для отключенной кнопки */
    color: #6c6c6c;              /* Серый текст для отключенной кнопки */
}
</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="generatePage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="generatingButton">
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="managePage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>manage</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="categoriesPage">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>categories</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="checkPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>100</number>
        </property>
        <property name="topMargin">
         <number>100</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>check</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>settings</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
